src/skills-builder/skills-builder-modal/view-results/RelatedSkillsSelectableBoxSet.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  SelectableBox, Chip, Stack, useMediaQuery, breakpoints,
+} from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import messages from './messages';
+
+const RelatedSkillsSelectableBoxSet = ({ jobSkillsList, selectedJobTitle, onChange }) => {
+  const { formatMessage } = useIntl();
+  const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth });
+
+  const renderTopFiveSkills = (skills) => {
+    const topFiveSkills = skills.sort((a, b) => b.significance - a.significance).slice(0, 5);
+    return (
+      topFiveSkills.map(skill => (
+        <Chip key={skill.id}>
+          {skill.name}
+        </Chip>
```

Comment:
Would it be difficult to add a test for this component/functionality?

Reply:
Should we be coding more protectively around this functionality? Do we need to ensure there are more than 5 entries? Is it possible that we run into any errors if we try to slice an array with < 5 items in it?

Reply:
(I'm messing around in a JSFiddle sandbox and it looks like maybe this won't throw an exception if we try to slice() beyond the size of the array... so maybe this isn't as much of an issue as I thought it would be)

Reply:
Yes, I don't think the `slice` will cause any issues for us here.

In regards to the first question, it would be nice to have a test to check the order of skills, ensuring that the ones with highest significance are placed at the top